### DISCUSSION TITLE
Pinch zoom for images

### DISCUSSION THREAD

**Discussion by rohanvakharia** [community/community#4073] [Mobile] [answered]
As the mobile screens are small, when we visit the .md pages, direct images, etc which you allow to see as a enduser can not be able to see the images in detail. If you allow to pinch zoom the images it would be a great improvement in the image preview feature.

**Comment by aconsuegra** [comment, marked as answer]
Thank you so much for the request! We are tracking being able to pinch to zoom internally and I added your note to our issue. In the meantime, please let us know if you have any other bugs or feature requests. We really appreciate it!

**Reply by halr9000** [reply]
Any news on this old request? I love that readme & code listings respect the OS font size or can even be overridden. That's great for accessibility, where mobile is an awesome enabler for those with vision disabilities. 

Now please do the same for images!

**Comment by htcfreek** [comment]
That's a very needed feature also when previewing image files in PR file lists.

**Comment by gtirloni** [comment]
Is this still being considered after 2y? It makes using the Android app very difficult on a daily basis because I have to open the browser, go to GitHub, find the repo and the image I wanted to see in the app in the first place.

**Reply by lostdoco** [reply]
@gitstua

**Comment by ghost** [comment]
> Thank you so much for the request! We are tracking being able to pinch to zoom internally and I added your note to our issue. In the meantime, please let us know if you have any other bugs or feature requests. We really appreciate it!

**Comment by twig** [comment]
Happy third birthday to this feature request
